github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -93,3 +93,35 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-okd.sh"
+    - name: pull-kubernetes-nmstate-e2e-ocp
+      skip_branches:
+        - release-\d+\.\d+
+      annotations:
+        fork-per-release: "true"
+      always_run: false
```

Comment:
true ?

Reply:
It takes a lot of time. Let's just trigger it manually when a PR is ready. It is also "optional" until we make sure it is stable enough.

Reply:
fair enough